pkg/virtctl/imageupload/imageupload.go

```diff
@@ -386,6 +389,12 @@ func (c *command) run(args []string) error {
 		return err
 	}
 
+	if dataSource {
+		if err = createDataSource(virtClient, name, namespace); err != nil {
```

Comment:
```suggestion
		if err := createDataSource(virtClient, name, namespace); err != nil {
```

You could use err scoped only to this if block here.